drivers/input/input_chsc5x.c

```diff
@@ -170,6 +167,62 @@ static int chsc5x_init(const struct device *dev)
 		k_msleep(1);
 	}
 
+	return 0;
+}
+
+#ifdef CONFIG_PM_DEVICE
+static int chsc5x_pm_action(const struct device *dev, enum pm_device_action action)
+{
+	const struct chsc5x_config *config = dev->config;
+	int ret;
+
+	if (config->reset_gpio.port == NULL) {
+		return -ENOTSUP;
+	}
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		ret = chsc5x_reset(dev);
+		break;
+
+	case PM_DEVICE_ACTION_SUSPEND: {
+		const uint8_t write_buffer[] = {
+			CHSC5X_BASE_ADDR1,
+			CHSC5X_BASE_ADDR2,
+			CHSC5X_BASE_ADDR3,
+			CHSC5X_ADDRESS_MODE,
+			0xF7, 0x16, 0x05, 0x00, 0x01, 0x00, 0x00, 0x00,
+			0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0xE9,
```

Comment:
maybe a comment line explaining what these means or where they are from?

Reply:
Sure, it's from a non-public datasheet, not sure if that's helpful? 😅

Reply:
yeah, no it does not help much :-) I mean, does the super duper secret datasheet give any other explanation? Like, reset all register to default or something? or is it just "send this and don't ask"?

Reply:
Ok, as usual with these hyper secret datasheet, there's a reason why they want to keep it secret... alright guess that'll do.